app/src/main/java/com/keyneez/presentation/main/id/IdFragment.kt

```diff
@@ -3,22 +3,55 @@ package com.keyneez.presentation.main.id
 import android.content.Intent
 import android.os.Bundle
 import android.view.View
+import android.widget.ImageView
+import androidx.fragment.app.viewModels
+import com.google.android.material.bottomsheet.BottomSheetDialog
 import com.keyneez.presentation.main.id.dialog.IdBenefitFragment
-import com.keyneez.presentation.main.id.dialog.IdProfileFragment
 import com.keyneez.presentation.ocr.guide.OcrGuideActivity
 import com.keyneez.util.binding.BindingFragment
 import com.keyneez.util.extension.setOnSingleClickListener
 import com.lab.keyneez.R
+import com.lab.keyneez.databinding.BotSheetIdProfileBinding
 import com.lab.keyneez.databinding.FragmentIdBinding
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class IdFragment : BindingFragment<FragmentIdBinding>(R.layout.fragment_id) {
+
+    private lateinit var bottomSheetBinding: BotSheetIdProfileBinding
+    private lateinit var bottomSheetDialog: BottomSheetDialog
+    private val viewModel by viewModels<IdViewModel>()
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding.vm = viewModel
         initIdLayout()
+        initBottomSheet()
+        initIdPhotoBtnClickListener()
+        initIdBackGround()
+    }
+
+    private fun initIdBackGround() {
+        if (viewModel.userData.value?.userCharacter?.rem(5) == 1) {
+            // 문화인-파란색
+            binding.ivIdMainBackground.setImageDrawable(R.mipmap.card_bg_mint)
+        } else if (viewModel.userData.value?.userCharacter?.rem(5) == 2) {
+            // 진로탐색러-초록색
+            binding.ivIdMainBackground.setImageDrawable(R.mipmap.card_bg_green)
+        } else if (viewModel.userData.value?.userCharacter?.rem(5) == 3) {
+            // 탐험가-핑크색
+            binding.ivIdMainBackground.setImageDrawable(R.mipmap.card_bg_pink)
+        } else if (viewModel.userData.value?.userCharacter?.rem(5) == 4) {
+            // 경제인-빨간색
+            binding.ivIdMainBackground.setImageDrawable(R.mipmap.card_bg_red)
+        } else {
+            // 봉사자-보라색
+            binding.ivIdMainBackground.setImageDrawable(R.mipmap.card_bg_purple)
+        }
```

Comment:
```suggestion
       when(viewModel.userData.value?.userCharacter?.rem(5)) {
              1 -> { ... }
              2 -> { ... }
              3 -> { ... }
              어쩌구
       } 
```

when 문으로 써보면 좋을 것 같아요!